### PR TITLE
Validation defaults + fix context

### PR DESCRIPTION
**Pull request by etokareva**

### CODE REVIEW

File: src/core/edit/edit.cell.view.js [hunk 1]
```diff
@@ -109,15 +109,19 @@ export class EditCellView extends View {
 				canExecute: cell => {
 					cell = cell || this.editor.cell;
 					const context = this.contextFactory(cell);
-					const key = context.column.key;
-					const validator = validationService.createValidator(model.validation().rules, key);
+					let isValid = true;
+					if (context){
```
Comment (vkorolev) [inline src/core/edit/edit.cell.view.js]: Space is missing  `) {`

File: src/core/edit/edit.cell.view.js [hunk 2]
```diff
@@ -260,6 +264,9 @@ export class EditCellView extends View {
 	}
 
 	contextFactory(cell, value, label, tag) {
+		if (!cell) {
```
Comment (klumba12) [inline src/core/edit/edit.cell.view.js]: this is not right context should always return context not boolean, if cell is undefined createContext shouln't be called

File: src/core/edit/edit.cell.view.js (hunk 1 above)
Comment (etokareva) [reply src/core/edit/edit.cell.view.js]: :-)

File: src/core/edit/edit.cell.view.js [hunk 3]
```diff
@@ -108,16 +108,18 @@ export class EditCellView extends View {
 				shortcut: this.shortcutFactory('commit'),
 				canExecute: cell => {
 					cell = cell || this.editor.cell;
-					const context = this.contextFactory(cell);
-					const key = context.column.key;
-					const validator = validationService.createValidator(model.validation().rules, key);
-					return cell
+					const isEditableCell = cell
```
Comment (klumba12) [inline src/core/edit/edit.cell.view.js]: rename to canEdit

**Review by klumba12 — APPROVED** [review]

**Pull request merged**